app/Exceptions/Handler.php

```diff
@@ -56,6 +58,9 @@ public function report(Exception $e)
      */
     public function render($request, Exception $e)
     {
+        // @todo we should try app('request') first but we can't guarantee its been created
```

Comment:
Does this happen when multisite calls `abort()` before everything is actually started?

Reply:
Yeah, the request is not fully formed at that stage.